### PR TITLE
ensure that vector3's are cast to arrays inside of spatial calculations

### LINKED ISSUE #10: casting of Vector3's in computations
We need to be careful with the `vector3` properties when using them in computations. When multiplying by an array, we should end-up with a numpy array, not a vector3. (e.g. 1919d1c0c113ea908f58b6094d545b4f2e847778). We should cast the location and orientation to numpy arrays upon the first computation (e.g. in calculating the vector potential in the recent pr #9)

### PR DESCRIPTION
**Pull request by lheagy**
- in the em base, ensure that vector3 properties like the orientation and location of a dipole are cast to arrays before calculations so that the output is always an array 
- add tests to ensure output of the magnetostatic calculations are arrays

fixes #10

### CODE REVIEW

File: tests/test_em_static.py [hunk 1]
```diff
@@ -60,6 +60,12 @@ def test_vector_potential(self):
             a_clws = self.clws.vector_potential(mesh.gridCC)[inds]
             a_mdws = self.mdws.vector_potential(mesh.gridCC)[inds]
 
+            self.assertTrue(isinstance(a_clws, np.ndarray))
+            self.assertFalse(isinstance(a_clws, properties.Vector3))
```
Comment (lheagy) [inline tests/test_em_static.py]: this tests that the output is an array and not a vector3

File: geoana/em/static.py [hunk 2]
```diff
@@ -255,10 +255,10 @@ def vector_potential(self, xyz, coordinates="cartesian"):
 
         # rotate the points to aligned with the normal to the source
         A = spatial.rotate_points_from_normals(
-            A, np.r_[0., 0., 1.], self.orientation, x0=self.location
+            A, np.r_[0., 0., 1.], np.array(self.orientation), x0=self.location
```
Comment (lheagy) [inline geoana/em/static.py]: - [ ] np.array(self.location)
- [ ] inside `rotate_points_from_normals`: ensure x0, orientation are numpy arrays

File: geoana/em/static.py [hunk 3]
```diff
@@ -255,10 +255,11 @@ def vector_potential(self, xyz, coordinates="cartesian"):
...
         # rotate the points to aligned with the normal to the source
         A = spatial.rotate_points_from_normals(
-            A, np.r_[0., 0., 1.], self.orientation, x0=self.location
+            A, np.r_[0., 0., 1.], np.array(self.orientation),
+            x0=np.array(self.location)
         )
 
         if coordinates.lower() == "cylindrical":
-            A = spatial.cartesian_2_cylindrical(np.array(xyz), A)
+            A = spatial.cartesian_2_cylindrical(xyz, A)
```
Comment (sgkang) [inline geoana/em/static.py]: as long as this outputs ndarray, I am fine.

**Comment by lheagy** [comment]
thanks @sgkang!

**Pull request merged**